tests/test_solvers/test_harbor_solver.py

```
@@ -1125,6 +1151,120 @@ async def test_flush_patch_timeout_writes_trajectory_no_marker(self, tmp_path, m
+        runner.write_text(f'{patched}\n\nif __name__ == "__main__":\n    main()\n')
+
+        env = os.environ.copy()
+        env["NEL_TEST_TRAJECTORY_PATH"] = str(traj_path)
+        proc = subprocess.Popen(
+            [sys.executable, str(runner)],
+            env=env,
+            stderr=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            text=True,
+        )
+        time.sleep(0.2)
```

Ruff (0.15.21)

[warning] Async functions should not call `time.sleep`

(ASYNC251)